controller.go

```diff
@@ -58,7 +58,7 @@ import (
 	"github.com/docker/libnetwork/driverapi"
 	"github.com/docker/libnetwork/hostdiscovery"
 	"github.com/docker/libnetwork/netlabel"
-	"github.com/docker/libnetwork/sandbox"
+	"github.com/docker/libnetwork/platform"
```

Comment:
IMO, it makes it very clear when you read a variable of type `platform.Sandbox`. You known what it is.